appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -0,0 +1,112 @@
+#Reads in SARIF files and rewrite absolute URIs to reference the source control version
+
+$logFilePath=$args[0]
+$repoProvider=$env:APPVEYOR_REPO_PROVIDER
+$projectSlug=$env:APPVEYOR_PROJECT_SLUG
+$repoName=$env:APPVEYOR_REPO_NAME
+$repoCommit=$env:APPVEYOR_REPO_COMMIT
+
+$builder = New-Object System.UriBuilder
+
+if($repoProvider -eq "gitHub"){
+  $builder.Scheme = 'https'
+  $builder.Host = 'raw.githubusercontent.com'
+}
+else{
+  throw "Repository provider i.e. gitHub, bitbucket ... is not recognized"
+}
```

Comment:
1. Don't mention bitbucket if the script doesn't support it.
2. Do tell the user how to fix the problem. For example:

```
throw "Unrecognized repository provider: $repoProvider. Set the APPVEYOR_REPO_PROVIDER environment variable to a supported provider. Supported providers are: github."
```